src/gui/ar/groundPlaneAnchors.js

```diff
@@ -177,26 +136,32 @@ createNameSpace("realityEditor.gui.ar.groundPlaneAnchors");
     // helper function to create the geometry for a surface anchor, including its X-Z axis handles
     function createAnchorGroup(frameKey) {
         const THREE = realityEditor.gui.threejsScene.THREE;
-        const group = new THREE.Group();
+
+        let originSize = 100;
+        const group = new THREE.Mesh(new THREE.BoxGeometry(originSize, originSize, originSize), new THREE.MeshBasicMaterial({color: originColor}));
         group.name = getElementName(frameKey) + '_group';
-        group.matrixAutoUpdate = false; // this is needed to position it directly with matrices
         group.visible = isPositioningMode;
-        let originSize = 100, axisSize = 50;
-        const originBox = new THREE.Mesh(new THREE.BoxGeometry(originSize, originSize, originSize),new THREE.MeshBasicMaterial({color: originColor}));
-        originBox.name = getElementName(frameKey) + '_originBox';
-        const xBox = new THREE.Mesh(new THREE.BoxGeometry(axisSize, axisSize, axisSize),new THREE.MeshBasicMaterial({color: xBoxColor}));
-        xBox.name = getElementName(frameKey) + '_xBox';
-        // const yBox = new THREE.Mesh(new THREE.BoxGeometry(axisSize, axisSize, axisSize),new THREE.MeshBasicMaterial({color:0x00ff00}));
-        // yBox.name = getElementName(frameKey) + '_yBox';
-        const zBox = new THREE.Mesh(new THREE.BoxGeometry(axisSize, axisSize, axisSize),new THREE.MeshBasicMaterial({color: zBoxColor}));
-        zBox.name = getElementName(frameKey) + '_zBox';
-        xBox.position.x = 150;
-        // yBox.position.y = 150;
-        zBox.position.z = 150;
-        group.add(originBox);
-        originBox.add(xBox);
-        // originBox.add(yBox);
-        originBox.add(zBox);
+
+        const options = {
+            size: realityEditor.device.environment.variables.transformControlsSize || 1,
+            hideY: true
+        }
+
+        let transformControl = realityEditor.gui.threejsScene.addTransformControlsTo(group, options, (e) => {
```

Comment:
nit: this could be simplified to `let transformControl = realityEditor.gui.threejsScene.addTransformControlsTo(group, options, onChange, onDraggingChanged);`